source/_components/light.mysensors.markdown

```diff
@@ -30,11 +30,11 @@ S_TYPE      | V_TYPE
 S_DIMMER    | [V_DIMMER\* or V_PERCENTAGE\*], [V_LIGHT\* or V_STATUS\*]
 S_RGB_LIGHT | V_RGB*, [V_LIGHT\* or V_STATUS\*], [V_DIMMER or V_PERCENTAGE]
 
-V_TYPES with a star (\*) denotes required V_TYPES. Use either V_LIGHT or V_STATUS and either V_DIMMER or V_PERCENTAGE for an applicable actuator.
+V_TYPES with a star (\*) denotes V_TYPES that should be sent with every update. I.e., for an S_DIMMER, send both a V_DIMMER/V_PERCENTAGE and a V_LIGHT/V_STATUS message.  For an S_RGB_LIGHT, send both a V_RGB and a V_LIGHT/V_STATUS message with a V_DIMMER/V_PERCENTAGE message being optional.  
```

Comment:
Hmm.  This was the issue we discussed here ( https://community.home-assistant.io/t/dimmable-rgb-lights-on-mysensors2-0-can-turn-off-cant-turn-on/8818/11 ).  I was seeing a problem when the device received a V_PERCENTAGE and sent back a V_PERCENTAGE acknowledgement without a V_STATUS.  Seems like you are saying the V_STATUS ack (on a V_DIMMER command) is only necessary when the dimmer value changes between zero and non-zero.  I'll test this.